test/cipher_tests.jl

```diff
@@ -97,9 +97,9 @@ criticalbytes = hex2bytes("6e6f74555446382855aa552de2888029")
 dummy = bytestring(decrypt(dec, encrypt(enc, criticalbytes)))
 @test isa(dummy, AbstractString)
 if !isdefined(Core, :String) || !isdefined(Core, :AbstractString)
-    @test !isa(dummy, ASCIIString)
+    @test !isa(dummy, String)
```

Comment:
Please at least have a look at what you are changing! This change doesn't even make sense....... Using an abstract type in precompile above is not likely to be a "nicer way" either.